functional/test/test_functional.py

```diff
@@ -800,6 +800,21 @@ class A(object):
         self.assertIsInstance(_wrap(A()), A)
         self.assert_type(self.seq(l))
 
+    def test_wrap_pandas(self):
+        # pylint: disable=superfluous-parens
```

Comment:
Not sure why this would be correctly throwing a lint error. Would you mind removing it, and if it turns out to be a lint error I disagree with I'll re-insert.